columnflow/tasks/framework/mixins.py

```diff
@@ -415,7 +418,7 @@ def events_used_in_training(self, dataset_inst: od.Dataset, shift_inst: od.Shift
         # shall be used in the training
         return (
             dataset_inst in self.ml_model_inst.used_datasets and
-            not shift_inst.x("disjoint_from_nominal", False)
+            not shift_inst.has_tag("disjoint_from_nominal")
```

Comment:
This changed from `not disjoint_from_nominal = False` to `not disjoint_from_nominal` so the logic changed. Is that correct?

Reply:
That `False` was just the default when the aux entry did not exist, but the logic still was "if not disjoint from nominal".